src/acore-wp-plugin/src/Components/ServerInfo/ServerInfoApi.php

```
@@ -15,12 +15,581 @@ public static function AccountCount() {
+   register_rest_route( ACORE_SLUG . '/v1', 'request-email-2fa', array(
+       'methods'             => 'POST',
+       'permission_callback' => function() { return is_user_logged_in(); },
+       'callback'            => function() {
+           $user    = wp_get_current_user();
+           $primary = get_user_meta($user->ID, 'wp_2fa_enabled_methods', true);
+           $methods = is_array($primary) ? $primary : ($primary !== '' ? [$primary] : []);
+           if (!in_array('email', $methods, true))
+               return new \WP_Error('no_email_2fa', __('Email 2FA is not enabled on your account.'), ['status' => 400]);
+
+           $code = (string) wp_rand(100000, 999999);
+           set_transient('acore_2fa_email_code_' . acore_2fa_unlock_key($user->ID), wp_hash($code), 10 * MINUTE_IN_SECONDS);
```

PHPStan (2.2.2)

[error] Constant MINUTE_IN_SECONDS not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(constant.notFound)